apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,26 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
+    if (
+      experiments.isEnabled('ai-rubrics') &&
+      hasScriptData('script[data-rubricdata]')
+    ) {
+      const rubricData = getScriptData('rubricdata');
+      const script = document.querySelector('script[data-level]');
+      const config = JSON.parse(script.dataset.level);
+      const teacherId = getStore().getState().currentUser.userId;
```

Comment:
Tess had asked for `teacherId` to be sent through - this was the only way I figured how to do it but again am open to feedback if there is a better way to go about this.

Reply:
I think this is okay (or at least I don't know of a better way)